src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
 'use strict';
 
-// write code here
+const fieldBody = document.querySelector('.field tbody');
+
+const appendRow = document.querySelector('.append-row.button');
+const removeRow = document.querySelector('.remove-row.button');
+const appendColumn = document.querySelector('.append-column.button');
+const removeColumn = document.querySelector('.remove-column.button');
+
+const minLength = 2;
+const maxLength = 10;
+
+let countRows = document.querySelectorAll('tr').length;
```

Comment:
Consider moving the query selector inside the countRows definition to avoid a potential reference error, like `let countRows = fieldBody.querySelectorAll('tr').length;`.